setup.py

```diff
@@ -56,6 +56,7 @@
             'hostname = rocker.extensions:Hostname',
             'name = rocker.extensions:Name',
             'network = rocker.extensions:Network',
+            'ipc = rocker.extensions:IPC',
```

Comment:
Please insert this in alphabetical order for ease of maintenance and readability.

Reply:
Added the basic unit test for high coverage and order the code in alphabetical order.